src/yb/rocksdb/util/thread_posix.cc

```diff
@@ -176,7 +176,11 @@ void ThreadPool::StartBGThreads() {
     CHECK_OK(yb::Thread::Create(
         category_name, std::move(thread_name),
         [this, tid]() { this->BGThread(tid); }, &thread));
-    bg_wait_states_.push_back(std::make_shared<yb::util::WaitStateInfo>());
+    bg_wait_states_.push_back(std::make_shared<yb::util::WaitStateInfo>(
+                                yb::util::AUHMetadata{
+                                    .top_level_node_id = "rocksdb",
+                                    .top_level_request_id = {0, 2},
+                                    .query_id = -2}));
```

Comment:
same comment as above

Reply:
Also since these metadata fields (along with client node ip) will be same for all the threads, can we just set them in the response PB instead of here?

Reply:
We are already maintaining WaitStateInfo per thread as bg_wait_states_ . Why leave it empty and handle this specially elsewhere? Better to update it to reflect the metadata for the items handled in these threads.